app/experimenter/templates/experiments/list.html

```diff
@@ -110,6 +110,9 @@ <h5>{{ experiment.population }}</h5>
               {% if experiment.enrollment_end_date %}
                 <p>Enrolling until {{ experiment.enrollment_end_date }}</p>
               {% endif %}
+              {% if experiment.survey_required %}
+                <span class="badge badge-secondary mb-2">Includes Survey</span>
+              {% endif %}
             </div>
```

Comment:
what's the margin bottom for? `mb-2`

Reply:
Visually I think it looks better

without
![Screen Shot 2019-07-17 at 3 53 30 PM](https://user-images.githubusercontent.com/25379936/61417157-1f544080-a8ab-11e9-9ac3-5e4b32889464.png)

with
![Screen Shot 2019-07-17 at 3 53 44 PM](https://user-images.githubusercontent.com/25379936/61417171-267b4e80-a8ab-11e9-9198-5741e3bac534.png)

Reply:
oh yea definitely! I wasn't considering the situation where the description column had less height than the includes survey column!